### ISSUE TITLE
Find new Mastodon bot host

### ISSUE THREAD

**Issue by metavida** [rosszurowski/small-seasons#7]
Sadly the botsin.space host is shutting down, going read only mid-December 2024 https://muffinlabs.com/posts/2024/10/29/10-29-rip-botsin-space/

I’d love to be able to keep following small seasons on the Fediverse if possible! 

Let me know if you’d be interested in help finding a new host or exploring self hosting options. I’m not sure how much time I’ll have but am glad to help how I can.

**Comment by rosszurowski** [comment]
Ah yes, I got that email recently. I'll find a new bot-friendly host and move it over.

I'm not super familiar with Mastodon — do you know if there's an easy way to port accounts over to a new server so people don't have to re-follow it on a new server? I might just make an announcement from the account if there's no better way.

**Comment by metavida** [comment]
Yes! There is a process for migrating your followers between servers when you move! This site has detailed instructions: https://fedi.tips/transferring-your-mastodon-account-to-another-server/

**Comment by rosszurowski** [comment]
Ok, I think I've shifted it to https://mastodon.social/@smallseasons. I'll monitor over the next couple days to see if the followers successfully transfer over. Thanks for sharing that guide — really helpful!

**Comment by metavida** [comment]
Looks like the follower transfer is starting to work! Confirmed that my account is following the bot on mastodon.social 

Now we just need to wait until the next small season to arrive!
